webapp/views/wpt-results.js

```diff
@@ -404,9 +416,30 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
       // path => {type, file[, refPath]} simplification.
       manifest: Object,
       screenshots: Array,
+      selectedMetadata: {
+        type: Array,
+        value: [],
+        observer: 'clearSelectedCells',
+      },
+      selectedCells: {
+        type: Array,
+        value: [],
+      },
+      testPlural: {
+        type: String,
+        computed: 'computeTestPlural(selectedMetadata)',
+      },
+      isTriagable: {
+        type: Boolean,
+        observer: 'isTriagableUpdated',
+      }
     };
   }
 
+  isTriagableUpdated(isTriagable) {
+    this.reloadData();
```

Comment:
Why do we need to reload data here?

Reply:
It is because we have to turn on/off `on-click` to each cell when the value of `isTriagable` changes